global/css/plugins/theme/colors.js

```diff
@@ -4,138 +4,8 @@
  */
 module.exports = function () {
   return {
-    transparent: "transparent",
-    current: "currentColor",
-    inherit: "inherit",
-    // Primary Colors
-    // https://identity.stanford.edu/design-elements/color/primary-colors/
-    "cardinal-red": {
-      DEFAULT: "#8C1515", // Stanford main brand color
-      light: "#B83A4B",
-      dark: "#820000",
-    },
-    black: {
-      DEFAULT: "#2E2D29", // Default text color on light background
-      90: "#43423E",
-      80: "#585754",
-      70: "#6D6C69",
-      60: "#767674",
-      50: "#979694",
-      40: "#ABABA9",
-      30: "#C0C0BF",
-      20: "#D5D5D4",
-      10: "#EAEAEA",
-      true: "#000000", // For when a higher contrast is needed
-    },
-    white: "#FFFFFF",
-    "cool-grey": "#53565A",
-    // Web Interactive Colors
-    // https://identity.stanford.edu/design-elements/color/web/
-    "digital-red": {
-      DEFAULT: "#B1040E", // Same as old Bright Red in previous color palette
-      light: "#E50808", // Same as old Vivid Red in previous color palette
-      dark: "#820000", // Same as Dark Cardinal Red in previous color palette
-    },
```

Comment:
I'm removing all the colors that are already in core Decanter and leaving only the `dark-mode` color here.